recipes/nbstripout/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "nbstripout" %}
+{% set version = "0.2.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8ca27ab35204b8352b376c071ed4543696d1dbb5088ab93dd282b134fdffd8fb
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pytest-runner
+  run:
+    - python
+    - setuptools
```

Comment:
Assuming this is only require for the entrypoint. So please make the change above and drop `setuptools` from the `run` dependencies.